sw/device/silicon_creator/rom_ext/e2e/ownership/BUILD

```diff
@@ -432,6 +432,7 @@ _FLASH_PERMISSION_TESTS = {
             binaries = {
                 ":flash_regions": "flash_regions",
             },
+            changes_otp = True,
```

Comment:
Looks like this line is present in master, I guess the two branches diverged so much that a backport isn't feasible anymore?

Reply:
Actually that commit probably would cherry-pick okay... f3a8481cfc2e887c043d8ee3b491df00addfe9ea

I don't think it's worth changing the commits here though now CI has passed

Reply:
Well spotted though, another case of maintaining multiple branches being a pain